modules/virt-querying-metrics.adoc

```diff
@@ -66,12 +66,24 @@ Returns the total amount (in bytes) of the virtual machine's storage-related tra
 `kubevirt_vmi_storage_write_traffic_bytes_total`::
 Returns the total amount of storage writes (in bytes) of the virtual machine's storage-related traffic.
 
-.Example storage-related traffic query
+`kubevirt_vmsnapshot_disks_restored_from_source_total`::
+Returns the total number of virtual machine disks restored from the source virtual machine.
+
+`kubevirt_vmsnapshot_disks_restored_from_source_bytes`::
+Returns the amount of space in bytes restored from the source virtual machine.
```

Comment:
Should these have the same heading as the others above them (noting that this section starts with "The following queries can identify VMs that are writing large amounts of data:")? They seem like they're in a different category and could use their own subtitle (maybe something snapshot-related)

Reply:
Yep, they are. I added an ".Examples of storage snapshot data queries" heading as they differ from the traffic-related queires.